### DISCUSSION TITLE
Cascader级联选择 多选勾选时 次级菜单还是展现的之前的

### DISCUSSION THREAD

**Discussion by hellopanpan** [ant-design/ant-design#33879] [Ideas]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/wk8ye)

### Steps to reproduce
1.点击一级Light，次级菜单展开
2.勾选LIght，
3.勾选Bamboo, 此时右侧还是Light的次级菜单

### What is expected?
第三步时右侧更新为Bamboo的次级菜单

### What is actually happening?
第三步时右侧更新LIght的次级菜单

| Environment | Info |
|---|---|
| antd | undefined |
| React | 17.0 |
| System | mac |
| Browser | chrome |

**Comment by afc163** [comment]
目前点击 Checkbox 的行为就是选择，不包含切换，我有点不确定修改为主楼里的方式是否更好。

**Comment by MadCcc** [comment]
> 目前点击 Checkbox 的行为就是选择，不包含切换，我有点不确定修改为主楼里的方式是否更好。

我觉得目前的是合理的，可以让社区自行讨论下
